babel.config.js

```diff
@@ -1,10 +1,19 @@
 module.exports = {
-    plugins: ["const-enum", "@babel/transform-typescript"],
+    plugins: [
+        "const-enum",
+        "@babel/transform-typescript",
+        [
+            "babel-plugin-emotion",
+            {
+                "cssPropOptimization": true
```

Comment:
[This is a default](https://emotion.sh/docs/@emotion/babel-plugin#csspropoptimization), I don't think it needs to be explicit